tacs/pymeshloader.py

```diff
@@ -894,3 +894,59 @@ def idMap(self, fromIDs, tacsIDDict):
                 return tacsIDDict[fromIDs]
             else:
                 return -1
+
+    @property
+    def struct_ids(self):
```

Comment:
Are struct ID's the same thing as the node ID's? Can we call them Node ID's to be consistent with the rest of the code?